openc3/lib/openc3/api/cmd_api.rb

```diff
@@ -140,9 +147,8 @@ def send_raw(interface_name, data, scope: $openc3_scope, token: $openc3_token)
     # @param target_name [String] Target name of the command
     # @param command_name [String] Packet name of the command
     # @return [Hash] command hash with last command buffer
-    def get_cmd_buffer(target_name, command_name, scope: $openc3_scope, token: $openc3_token)
-      target_name = target_name.upcase
-      command_name = command_name.upcase
+    def get_cmd_buffer(*args, scope: $openc3_scope, token: $openc3_token)
+      target_name, command_name = _extract_target_command_names('get_cmd_buffer', *args)
```

Comment:
Original goal with this PR was to allow various APIs to support both method signatures of `get_cmd_buffer("INST", "ABORT")` and `get_cmd_buffer("INST ABORT")`.